### PR TITLE
Roman-Style Pillar and Obelisk

### PR DESCRIPTION
**Pull request by ujjman**
This Pull Request adds roman style pillar and obelisk.

### CODE REVIEW

File: assets/prefabs/structures/marbleObelisk.prefab [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "StructureTemplate": {
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill": [
+        { "blockType": "AdditionalStructureTemplates:obeliskBase", "region": { "min": [1, 1, 1], "size": [1, 1, 1]}},
+        { "blockType": "AdditionalStructureTemplates:obeliskMiddle", "region": { "min": [1, 2, 1], "size": [1, 12, 1]}},
+        { "blockType": "AdditionalStructureTemplates:obeliskTop", "region": { "min": [1, 13, 1], "size": [1, 1, 1]}}
+]
+}
+}
```
Comment (Cervator) [inline assets/prefabs/structures/marbleObelisk.prefab:14]: Trivial: Newline at the end of each text file, brackets should be aligned with their openers. Something like:

```
    "SpawnBlockRegions": {
        "regionsToFill": [
        { "blockType": "AdditionalStructureTemplates:obeliskBase", "region": { "min": [1, 1, 1], "size": [1, 1, 1]}},
        { "blockType": "AdditionalStructureTemplates:obeliskMiddle", "region": { "min": [1, 2, 1], "size": [1, 12, 1]}},
        { "blockType": "AdditionalStructureTemplates:obeliskTop", "region": { "min": [1, 13, 1], "size": [1, 1, 1]}}
        ]
    }
} 
```

File: module.txt [hunk 2]
```diff
@@ -6,8 +6,11 @@
     "description" : "A collection of various structure templates.",
     "dependencies" : [
     	{"id": "Core", "minVersion": "2.1.1-SNAPSHOT"},
+    	{"id": "CoreAssets", "minVersion": "1.0.1"},
+    	{"id": "CoreBlocks", "minVersion": "1.0.1"},
     	{"id": "StructureTemplates", "minVersion": "0.3.0-SNAPSHOT"},
-    	{"id": "Fences", "minVersion": "0.1.1-SNAPSHOT"}
+    	{"id": "Fences", "minVersion": "0.1.1-SNAPSHOT"},
+    	{"id": "StructuralResources", "minVersion": "0.1.1-SNAPSHOT"}
```
Comment (Cervator) [inline module.txt]: This won't actually work: needs to be `2.0.0` which is the latest version of StructuralResources

**Review by Cervator — CHANGES_REQUESTED** [review]
Very minor changes needed, but I could get them to work in-game and they look nice. Well done :-)

**Review by ujjman — COMMENTED** [review]
files changed with aligned brackets

**Comment by ujjman** [comment]
I have aligned the brackets and changes minVer of StructuralResources to 2.0.0

**Comment by Cervator** [comment]
Thanks :-)

You had some tab characters instead of spaces sneak in (we use spaces, not tabs) and you dropped some starting brackets down to the next line which is another minor style inconsistency. But those are trivial so I reverted those bits while merging, see https://github.com/Terasology/AdditionalStructureTemplates/commit/4b98cd2e5878feffb8b8047f398f0755e9f8e018 if you're curious for the tiny details 👍

**Pull request merged**